### PR TITLE
Fixes 38 Surplus being unprintable in the autolathe and 38 speedloader description

### LINKED ISSUE #2814: [BUG] 38 Surplus isnt in the autolathe
## Reproduction: 
Check the autolathe. No surplus 38 ammo design. Weep.

### PR DESCRIPTION
**Pull request by Gristlebee**
## About The Pull Request

Fixes #2814 and makes 38 surplus ammo able to be printed from the autolathe. Changes the damage blurb on the 38 speedloader to reflect standard 38 ammo's new stats.
## Why It's Good For The Game
Bug fixes are good.

## Changelog

:cl:
fix: 38 Surplus ammo should be printable in the autolathe.
fix: 38 speedloader damage blurb is accurate to it's new stats.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -798,10 +798,10 @@
 	build_path = /obj/item/weaponcrafting/receiver
 	category = list("hacked", "Security")
 
-/datum/design/c38
+/datum/design/c38_surplus
 	name = "Ammo Box (.38 surplus)"
-	id = "c38"
-	build_type = AUTOLATHE | PROTOLATHE
```
Comment (west3436) [inline code/modules/research/designs/autolathe_designs.dm:804]: Stealth prolathe nerf - add this to the PR description

Comment (Gristlebee) [reply code/modules/research/designs/autolathe_designs.dm:804]: Added.

I'll also note #2775 already touched on this and restricted 38 ammo to the security protolathe only. No protolathe is also currently able to print surplus ammo either.

Comment (Gristlebee) [reply code/modules/research/designs/autolathe_designs.dm:804]: Turns out Im insane and missed the fact the other surplus ammo types also have protolathe print flags. Readded the flag for consistency.

**Comment by Gristlebee** [comment]
Updated the speedloader desc while I was at it since 38 now has the same stats as 9mm. Changed the PR description to reflect that I added back the protolathe flag. Everything should be in order.

**Review by MarkSuckerberg — APPROVED** [review]
good catch

**Comment by Imaginos16** [comment]
Insane

**Pull request merged**